src/meta/src/rpc/ddl_controller.rs

```diff
@@ -519,19 +576,161 @@ impl DdlController {
         }
     }
 
+    async fn inject_replace_table_job(
```

Comment:
Please add some strdoc for it.

Reply:
Can we change this logic as a last part of build streaming job if replace info provided?